### ISSUE TITLE
Remove warning

### ISSUE THREAD

**Issue by NikKovIos** [nerzh/swift-telegram-sdk#37]
In code there is a warning: `#warning("THIS CODE FOR FAST FIX, BECAUSE https://github.com/vapor/multipart-kit/issues/63 not accepted yet")`. This MR already merged. How about to remove it?

**Comment by nerzh** [comment]
Yes, I know, but if this code works now, do you really want to change it? If so, you can fix it and make a pull request for it, but be sure to check all file methods after that

**Comment by nerzh** [comment]
ok ...

**Comment by NikKovIos** [comment]
Нет достаточной экспертности чтобы править этот ворнинг. 

Тем не менее ворнинг явно говорит что он уже не актуален.
